doc/sphinx/source/ops/pad.rst

```diff
@@ -28,15 +28,15 @@ Inputs
 Attributes
 ----------
 
-+-----------------------+-------------------------------------------------------+
-| Name                  | Description                                           |
-+=======================+=======================================================+
-| ``padding_below``     | Shape of padding added before ``arg``                 |
-+-----------------------+-------------------------------------------------------+
-| ``padding_above``     | Shape of padding added after ``arg``                  |
-+-----------------------+-------------------------------------------------------+
-| ``padding_interior``  | Shape of padding inserted between elements of ``arg`` |
-+-----------------------+-------------------------------------------------------+
++-----------------------+---------------------------------------------------------------------+
+| Name                  | Description                                                         |
++=======================+=====================================================================+
+| ``padding_below``     | Padding added before ``arg``. May be negative.                      |
++-----------------------+---------------------------------------------------------------------+
+| ``padding_above``     | Padding added after ``arg``. May be negative.                       |
++-----------------------+---------------------------------------------------------------------+
+| ``pad_mode``          | Padding mode: CONSTANT(default), EDGE or REFLECT.                   |
++-----------------------+---------------------------------------------------------------------+
```

Comment:
To indicate that `CONSTANT`, `EDGE`, and `REFLECT` are the options here, might do a text variant of some sort, bold or monospace